docker/Makefile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: push
+
+TAG = 0.1
+PREFIX = quay.io/coffeepac/kafka-monitor
```

Comment:
quay.io uses GitHub for auth and namespaces based on GitHub user ID.  I don't think we can push to /default/ like that.  If I can't do something like you suggest do you have a group GitHub account you can make a quay repo for?  Or I can push it to somewhere under quay.io/samsung_cnct/

I'll take a look at having the makefile do the bashy have a default value if not set thing.  Should be just one line.